module3/lessons/Week5/Networking.md

```diff
@@ -0,0 +1,129 @@
+
+* Online communication
+* Meeting people in person
+* 1:1 interactions
+* Casual group conversations
+* Seeking people out on your own
+* Getting introduced by someone else (an event buddy)
+
+Once you are able to identify what energizes you, lean into it! Find ways to capitalize on this energy. Here are some examples:
+
+### Example #1: Networking through social media
+
+If you feel energized by engaging in social networking over Twitter or LinkedIn, here are some tips to maximize the impact of the interactions:
+* Build a robust profile to allow others to get to know you. Note: you’ll learn more about this next week, but here are some [resources for personal branding](https://careerdev.turing.edu/resources/branding_resources).
+* Seek out people who share your interests. Interested in artificial intelligence? Block chain? Diversity, equity, and inclusion in tech? Find your people. Check out hashtags, influential people or companies in your field, and other current conversation threads to join in.
+* Don’t leave one-off comments or “likes.” Take time to engage in conversation, listen to others, share helpful resources – just like you would in any conversation.
+
+### Example #2: Networking at organized events
+
+If you feel energized by meeting new people with shared interests at an event that brings you together, here are some tips to maximize the impact of the interactions:
+* Circulate the room. Make sure you don’t only talk to people you already know. Find people you haven’t met yet and introduce yourself.
+* Share your story. Having a concise pitch to describe who you are and what your goals are in this new career will be a helpful way to connect with others right away.
+* Ask questions and practice active listening. When you are able to get someone talking about something they are excited about, it can lead to deeper connections.
+* Follow up. Make sure to get contact information to have another conversation and reference what you talked about when you met.
+
+Even though you’ll lean into your comfort zone, you can also prepare for what to do when you leave them. If you were to engage in a networking activity that drains you, how could you recover? Take time to plan an activity beforehand and afterwards that works with your energy. If you’re an introvert, this might mean preparing for a large group networking event by having some alone time before and after the event, such as reading a book or taking a walk. Taking the time to prepare for this energy drain will help you be more fully yourself in the activity.
+
+Here is some additional reading on how to use your place on the introversion-extroversion spectrum to your advantage:
+* [Are you an Introvert or an Extrovert? What it Means for Your Career](https://www.fastcompany.com/3016031/are-you-an-introvert-or-an-extrovert-and-what-it-means-for-your-career)
+* [How Being an Introvert Helped Me in My First Job](https://quietrev.com/how-being-an-introvert-helped-me-in-my-first-job/)
```

Comment:
What are your thoughts about students completing a free personality assessment? There are a few decent ones out there that are maybe 10-15 minutes. I took a Myers-Briggs in college in Psych 101 and I live by those letters LOL (**INTJ**, if you were wondering/couldn't already tell). Here are a couple links:
- Myers-Briggs: https://www.16personalities.com/free-personality-test
- DiSC: https://discpersonalitytesting.com/free-disc-test/

Reply:
I like the idea of students doing more to learn about themselves - I'll add this activity in!  It also gives them a chance to do something - this is a lot of sit-and-get 😆